src/Config/SettingsTab.ts

```diff
@@ -114,6 +114,19 @@ export class SettingsTab extends PluginSettingTab {
         containerEl.createEl('h4', { text: 'Date Settings' });
         // ---------------------------------------------------------------------------
 
+        new Setting(containerEl)
+            .setName('Set created date on every added task')
+            .setDesc(
+                "Enabling this will add a timestamp ➕ YYYY-MM-DD at the end when a task is created with 'Create or edit task'",
```

Comment:
I didn't notice this before, but it says the new date will be added at the end, and in fact it isn't the end.
I'm a bit stuck on the wording, because the obvious alternative, 'beginning' isn't true either.

I might have said 'before all emojis' but there is about to be a non-emoji based format added.

Maybe 'before the other 'date signifiers'?

Reply:
Or 'date values'?